xarray/backends/h5netcdf_.py

```diff
@@ -117,13 +118,16 @@ def open(
         lock=None,
         autoclose=False,
         invalid_netcdf=None,
+        phony_dims=None,
     ):
         import h5netcdf
 
         if format not in [None, "NETCDF4"]:
             raise ValueError("invalid format for h5netcdf backend")
 
         kwargs = {"invalid_netcdf": invalid_netcdf}
+        if LooseVersion(h5netcdf.__version__) >= LooseVersion("0.8.0"):
+            kwargs["phony_dims"]: phony_dims
```

Comment:
Should this be
```suggestion
            kwargs["phony_dims"] = phony_dims
```
?

Reply:
@mathause thanks for spotting, you're right.